### ISSUE TITLE
Accessibility: Category and Tag Labels

### ISSUE THREAD

**Issue by wrenaria** [UCDavisLibrary/ucdlib-theme-wp#329]
Link labels need to be unique for screenreaders and provide context for the link. 

Ideally, I think we try not to repeat tags and category text on news posts, but it does happen now and then. I am wondering if it would be worth adding an aria label to append the section label to the beginning of those button links, especially since there are also possible instances where the words used as tags/categories are used as link text for another purpose in the body text of the news posts.  

For example, the aria text could be `category: food and wine` and `tag: food and wine`

Even without repetition in the content, these labels would provide beneficial context to the user. 

[Example of post with matching category and tag labels](https://library.ucdavis.edu/news/beaulieu-vineyard-donates-historic-archives-to-uc-davis-library/)

<img width="834" height="345" alt="Image" src="https://github.com/user-attachments/assets/2881bbce-6ebe-4fce-95da-c35c4b76c277" />

Related: _[Site Improve Report](https://my2.siteimprove.com/Inspector/728315/A11Y/Page?pageId=313254294353&impmd=130BA75F00FDF9C4E13726DF81FAB862&expand=potential&issue=cantTell+failed&wcag=two#/sia-r81/cantTell)_

**Comment by spelkey-ucd** [comment]
Released [v4.4.0](https://github.com/UCDavisLibrary/ucdlib-theme-wp/releases/tag/v4.4.0)

**Comment by spelkey-ucd** [comment]
I messed up this implementation a little. `aria-label` attributes are getting double escaped, which is causing a mismatch with the visible text for some tags: [site improve report](https://my2.siteimprove.com/Accessibility/728315/NextGen/Issue/1?ruleId=14&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=20&siteTarget.wcagVersion=20&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

chatgpt says i need to do the escaping on the attribute itself and not on the variable:
```twig
{% if text and href %}
  {% if kwargs.term.taxonomy %}
    {% set ariaLabel = kwargs.term.taxonomy ~ ': ' ~ text %}
  {% else %}
    {% set ariaLabel = 'Site Content Related to ' ~ text %}
  {% endif %}

  <a href="{{ href }}" class="tags__tag" aria-label="{{ ariaLabel|e('html_attr') }}">{{ text }}</a>
{% endif %}

```

**Comment by spelkey-ucd** [comment]
[Released v4.5.0](https://github.com/UCDavisLibrary/ucdlib-theme-wp/releases/tag/v4.5.0)
